test/Table_Tests/src/Database/Postgres_Spec.enso

```diff
@@ -21,7 +21,65 @@ import project.Database.Helpers.Name_Generator
 import project.Common_Table_Spec
 import project.Aggregate_Spec
 
-postgres_specific_spec connection pending =
+postgres_specific_spec connection db_name pending =
+    Test.group "[PostgreSQL] Schemas and Databases" <|
+        Test.specify "should be able to get current database and list databases" <|
+            connection.database . should_equal db_name
+            connection.databases.length . should_not_equal 0
+            connection.databases.contains db_name . should_be_true
+            Meta.is_same_object connection (connection.set_database db_name) . should_be_true
+
+        Test.specify "should be able to get current schema and list schemas" <|
+            connection.schema . should_equal "public"
+            connection.schemas.length . should_not_equal 0
+            connection.schemas.contains "public" . should_be_true
+            connection.schemas.contains "information_schema" . should_be_true
+            Meta.is_same_object connection (connection.set_schema "public") . should_be_true
+
+        Test.specify "should allow changing schema" <|
+            connection.set_schema "information_schema" . schema . should_equal "information_schema"
+
+        databases = connection.databases.filter d->((d!=db_name) && (d!='rdsadmin'))
+        pending_database = if databases.length != 0 then Nothing else "Cannot tests changing database unless two databases defined."
+        Test.specify "should allow changing database" pending=pending_database <|
+            database = databases.first
+            connection.set_database database . database . should_equal database
```

Comment:
Can we somehow add a check here to really verify that we changed the database and not only some internal descriptor?

For example - can we shomehow verify that now we "don't see" the tables from the other database?

Otherwise the test checks that our internal state changed but it doesn't prove that we really are able to work with the changed database.